cfe_internal/enterprise/federation/federation.cf

```diff
@@ -316,6 +320,27 @@ bundle agent imported_data
         contain => default:in_shell;
 }
 
+bundle agent superhub_schema
+# @brief Run SQL script to migrate to superhub partitioned table schema
```

Comment:
Run the SQL to ensure the table has been migrated (small wording change to indicate the script is always run, but change is not always done.